boards/BMS/Src/Io/Io_PreCharge.c

```diff
@@ -0,0 +1,13 @@
+#include "Io_PreCharge.h"
+#include "main.h"
+
+void Io_PreCharge_Enable(void)
+{
+    HAL_GPIO_WritePin(PRE_CHARGE_EN_GPIO_Port, PRE_CHARGE_EN_Pin, GPIO_PIN_SET);
```

Comment:
I noticed some of the other Io methods that enable/disable pins return ExitCodes (like in Io_OkStatuses), whereas these return void. Is there reason to use one over the other?

Reply:
Aside from this question everything looks good

Reply:
For the fxns in Io_OkStatuses I think Ken wanted to differentiate pins that we could actually use to enable OkStatuses from ones that couldn't.

Reply:
In our case since we're only working with the PRE_CHARGE_EN pin (which also happens to be a pin we can control atm) using a void fxn makes the most sense

Reply:
ok

Reply:
ok